README.md

```diff
@@ -1,22 +1,56 @@
-- `2` - cannot unpack archive: not enough disk space
-- `3` - cannot unpack archive: any other reason
-- `4` - invalid checksum of db
-- `5` - cannot verify checksum for some reason
-- `6` - cannot create a backup file
-- `7` - invalid checksum of archive
-- `8` - cannot validate archive checksum
\ No newline at end of file
+# quicksync-rs
+
+When a new node joins the Spacemesh network, it must first get up to speed with the rest of its peers. This process is referred to as "syncing" and is a pre-requisite to running a full or a smeshing node. Historically, it has been difficult for smeshers to successfully sync their nodes owing to how time-consuming the syncing process is. Primarily, syncing includes downloading and independently verifying all blocks, transactions, ATXs, along with some other data, and reconstructing the current state from scratch. Naturally, this took the average smesher a lot of time. As such, in response to the growing difficulty of syncing a fresh node, we have prepared a way to speed up the syncing process. Introducing, Quicksync.
+
+With Quicksync, instead of performing all of the syncing actions as stated above and calculating the network state from genesis, one just needs to download the current state from a trusted peer like the Spacemesh dev team or some other node. While this runs contrary to the web3 philosophy of "Don't trust, verify", we believe that this could be a choice some smeshers may be interested in given the high rate of trouble with syncing. Moreover, nothing precludes a smesher from verifying this state in the background once it is downloaded.
+
+The state (also called an archive) that is downloaded is in the form of a state.sql file and can either be downloaded automatically using Smapp, or manually by using the `quicksync-rs` utility. 
+
+Instructions for using `quicksync-rs` to download the state are given below.
+
+## Windows
+
+1. Download the latest release of the `quicksync-windows-vX.X.X.zip` file from the GitHub releases section.
+2. Extract the `quicksync.exe` file from the zip file downloaded in step 1.
+3. Move the `quicksync.exe` file to your `go-spacemesh` folder.
+4. If you see a `state.sql` file in this folder, delete it. Otherwise, continue to step 5.
+5. Open the Windows Powershell in this folder. You can do this by holding the "shift" key, right-clicking, and selecting the "Open PowerShell here" option.
+6. Inside the powershell, type `.\quicksync.exe --help` and press enter. This will show you the available options.
+7. We want to download the state database. Type `.\quicksync.exe download --node-data <path-to-your-node-data-folder>`. Note that the node data folder can be identified by the other folders and files it contains such as `bootstrap/`, `p2p/`, `genesis.json`, etc. A lot of smeshers label the node data folder as `sm_data` or just `data`. In the official Spacemesh docs and guides, this folder has been labelled as `node_data`.
```

Comment:
Isn't it a `spacemesh/` directory in the user's home directory by default?

Reply:
Not sure, don't have a Windows PC. But don't these instructions cover all scenarios?

Reply:
Yeah, but a user might have a hard time locating `<path-to-your-node-data-folder>`. It would be nice to tell him what the default is.

Reply:
Agree. Do you know what the default path is for Windows and Linux?

Reply:
- Win: `C:\Users\{USERNAME}\AppData\Roaming\Spacemesh`
- Linux: `~/.config/Spacemesh`
- MacOS: `~/Library/Application\ Support/Spacemesh`

Reply:
Updated the instructions to make them simpler, added the default paths. Can you guys please take another look to ensure accuracy?

Reply:
And afaik there is NO defaults for node-data you need to explicitly state it when running.

cc @poszu @hasanza

Reply:
Actually it is set to default as https://github.com/spacemeshos/go-spacemesh/blob/develop/config/config.go#L44 which means `~/spacemesh` and equivalent on other systems.

Reply:
Makes sense, @pigmej 👍

Reply:
@pigmej @brusherru Just pushed the latest commit. Used `~/spacemesh` as the default dir for Linux/MacOS and `C:\Users\{USERNAME}\spacemesh` for windows. I double checked on MacOS and the default is correct. For windows, I looked around and found [this stackoverflow Q/A thread](https://stackoverflow.com/questions/78463337/go-os-userhomedir-returning-c-users-user-name-on-wsl) which more or less confirms the windows default as well.